db/migrate/20210614165400_rename_names_for_users.rb

```diff
@@ -0,0 +1,6 @@
+class RenameNamesForUsers < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :users, :lastname, :last_name
```

Comment:
@malparty Same as comment above, changes can be done to same migration instead of creating new migration, as the migration is not yet run on server.

Reply:
Yep, done! :)